test/integration/external_app/test_ext_objects.py

```diff
@@ -198,7 +198,7 @@ def setUp(self):  # pylint: disable=missing-docstring
     super(TestInternalAppObjects, self).setUp()
     self.api = api_helper.Api()
 
-  @ddt.data(*all_models.get_scope_models())
+  @ddt.data(*TestBaseExternalObjects.OBJECTS)
   def test_scoped_import_deprecated(self, model):
```

Comment:
Please fix the test method name as well since it is not testing only scoped objects anymore.
```suggestion
  def test_internal_object_import_deprecated(self, model):
```

Reply:
Done